examples/hydro/paramfile.genic

```diff
@@ -30,9 +30,9 @@ FileWithTransferFunction = ../camb_transfer_99.dat  # filename of transfer funct
 InputSpectrum_UnitLength_in_cm = 3.085678e24 # defines length unit of tabulated
                                                                    # input spectrum in cm/h. 
                                     # Note: This can be chosen different from UnitLength_in_cm
-    
 
-PrimordialIndex  = 1. # may be used to tilt the primordial index. Only used if WhichSpectum != 2
+PrimordialAmp = 2.215e-9
+PrimordialIndex  = 0.971 # may be used to tilt the primordial index.
```

Comment:
I don't think this is always used? Why is the Only used clause removed? We may want to rename the parameter to reflect that it doesn't always apply!

Reply:
Oops, yes, will fix.